apps/web-giddh/src/app/permissions/components/list/permission.list.component.ts

```diff
@@ -104,7 +104,7 @@ export class PermissionListComponent implements OnInit, AfterViewInit, OnDestroy
     }
 
     public updateRole(role: NewRoleClass) {
-        let data = new NewRoleClass(role.name, role.scopes, role.isFixed, role.uniqueName, true);
+        let data = new NewRoleClass(role.name, role.scopes, role.isFixed, role?.uniqueName, true);
```

Comment:
is this correct logic? If role is not set, won't it give error on name or scopes or isFixed ???